src/Build/BackEnd/Shared/BuildResult.cs

```diff
@@ -116,6 +116,8 @@ public class BuildResult : INodePacket, IBuildResults
         /// </summary>
         private ProjectInstance _projectStateAfterBuild;
 
+        private BuildRequestDataFlags _buildRequestDataFlags;
```

Comment:
Please don't forget adding `translator.TranslateEnum(ref _buildRequestDataFlags, (int)_buildRequestDataFlags)` to the `Translate` method.